tensorflow/tools/ci_build/builds/test_installation.sh

```diff
@@ -410,7 +407,7 @@ SKIP_COUNTER=0
 FAILED_TESTS=""
 FAILED_TEST_LOGS=""
 
-if [[ "${TF_BUILD_USE_GPU}" == "1" ]]; then
+if [[ "${IS_GPU}" == "1" ]]; then
```

Comment:
Where does IS_GPU come from? It's not documented any more with the other variables.